togetherly/lib/providers/chore_provider.dart

```diff
@@ -26,18 +29,26 @@ class ChoreProvider with ChangeNotifier {
   //   return personId == null ? null : choresAssignedToPerson(personId);
   // }
 
-  Future<void> addChore(Chore chore) async {
-    await _service.insertChore(chore);
+  Future<void> addChore(Chore chore, children) async {
```

Comment:
`children` should have its datatype specified.

Reply:
updated! it is supposed to be a list of children!